contents/docs/hogql/index.mdx

```diff
@@ -20,6 +20,8 @@ HogQL expressions can be used inside insights and dashboards as filters and/or b
 
```

Comment:
Same as previous PR: Can include hyperlink to docs directly

Also, I think this intro needs to be reworked a bit. Something along 

```
[HogQL expressions]((/docs/hogql/expressions) enable you to xyz. They can be used inside [insights](hyperlink) and [dashboards](maybe hyperlink also?) as filters and/or breakdown values. 

When filtering or breaking down your insight, select the "HogQL" tab, and add your expression.

```

Reply:
Got it